crates/forge_analyzer/src/definitions.rs

```diff
@@ -917,6 +918,7 @@ impl<'cx> FunctionAnalyzer<'cx> {
                 };
                 let first_arg = first_arg?;
                 let is_as_app = authn.first() == Some(&PropPath::MemberCall("asApp".into()));
+                let api_call = classify_api_call(first_arg);
```

Comment:
You can remove this, since this isn't being used anywhere.

Reply:
Done